src/__tests__/Button.test.tsx

```diff
@@ -91,8 +91,9 @@ describe('Button', () => {
   })
 
   it('styles icon only button to make it a square', async () => {
-    const container = render(<IconButton icon={SearchIcon} iconLabel="Search icon only button" />)
+    const container = render(<IconButton icon={SearchIcon} aria-label="Search button" />)
     const IconOnlyButton = await container.findByRole('button')
     expect(IconOnlyButton).toHaveStyleRule('padding-right', '8px')
+    expect(IconOnlyButton).toMatchSnapshot()
```

Comment:
I feel like aria attributes fall under logic, so I think it's worth adding a separate test for it like..

```
expect(IconOnlyButton).toHaveAttribute('aria-label', labelThingy)
```